deploy/marathon/resources/connected-car.json

```diff
@@ -40,18 +20,17 @@
       "disk": 10,
       "gpus": 0,
       "instances": 2,
-      "backoffSeconds": 1,
-      "backoffFactor": 1.15,
-      "maxLaunchDelaySeconds": 3600,
+      "upgradeStrategy": {
+        "maximumOverCapacity": 0,
+        "minimumHealthCapacity": 0.5
+      },
+      "killSelection": "YOUNGEST_FIRST",
       "env": {
-        "KAFKA_SERVICE_NAME": "broker.kafka.l4lb.thisdcos.directory:9092",
+        "RP_PLATFORM": "mesos",
+        "RP_NAMESPACE": "connected-car",
+
         "CASSANDRA_SERVICE_NAME": "_native-client._node-0-server._tcp.cassandra.mesos",
-        "DUMMYSERVICE_BIND_IP": "0.0.0.0",
-        "DUMMYSERVICE_BIND_PORT": "$PORT0",
-        "AKKA_REMOTING_HOST": "$HOST",
-        "AKKA_REMOTING_BIND_HOST": "0.0.0.0",
-        "AKKA_REMOTING_PORT": "$PORT1",
-        "AKKA_REMOTING_BIND_PORT": "$PORT1",
+        "KAFKA_SERVICE_NAME": "_broker._kafka-0-broker._tcp.kafka.mesos",
```

Comment:
Seems to be the broker address for my own DC/OS cluster, hopefully yours matches